src/cmd/tag.go

```diff
@@ -142,19 +137,51 @@ opslevel get tag --type=Team ID|ALIAS | jq 'from_entries'   # values of all keys
 
 		var output []opslevel.Tag
```

Comment:
If `var output []opslevel.Tag` is substituted with
`output := []opslevel.Tag{}` 
then the `if len(output) == 0 { ... }` chunk below can be removed

Reply:
nice! added this in latest commit.